doc/rados/configuration/osd-config-ref.rst

```diff
@@ -494,6 +494,40 @@ Operations
 :Default: ``5``
 :Valid Range: 1-63
 
+``osd snap trim sleep``
+
+:Description: Time in seconds to sleep before next snap trim op.
+              Increasing this value will slow down snap trimming.
```

Comment:
Should this mention that setting this value supersedes the settings below?

Reply:
I believe that is true for all the sleep values, added a commit to be more specific